src/Microsoft.Extensions.RuntimeEnvironment.Sources/RuntimeEnvironment.cs

```diff
@@ -9,15 +9,21 @@ internal static class RuntimeEnvironment
     {
         static RuntimeEnvironment()
         {
-            OperatingSystemVersion = Native.PlatformApis.GetOSVersion();
-
             RuntimeType = GetRuntimeType();
```

Comment:
Maybe move this to to the property too 
`static public string RuntimeType { get; } = GetRuntimeType();` and you can remove the static ctor.